src/commands/filterCommand.ts

```diff
@@ -0,0 +1,598 @@
+import * as vscode from "vscode";
+import { Logs } from "../models/logs";
+import {
+  commands
+} from "../utils/common/commands";
+import {
+  SeverityLevel,
+  StateLevel,
+  constants
+} from "../utils/common/constants";
+import { messages } from "../utils/common/messages";
+import { updateStateFilter } from "../utils/common/globalState";
+
+export class FilterCommand {
+  context: vscode.ExtensionContext;
+  logs: Logs;
+  private activeSeverities: SeverityLevel[] = [
+    SeverityLevel.high,
+    SeverityLevel.medium,
+  ];
+  private activeStates: StateLevel[] = [
+    StateLevel.confirmed,
+    StateLevel.toVerify,
+    StateLevel.urgent,
+    StateLevel.notIgnored,
+  ];
+  constructor(context: vscode.ExtensionContext, logs: Logs) {
+    this.context = context;
+    this.logs = logs;
+  }
+
+  public getAtiveSeverities() {
+    return this.activeSeverities;
+  }
+
+  public getActiveStates() {
```

Comment:
i think this should be private, and it the filters should be applied in this class